docs/backend/content-types/index.md

```diff
@@ -12,15 +12,220 @@ myst:
+
+  <property name="allow_discussion">False</property>
+  <property name="factory">Document</property>
+  <property name="icon_expr">string:contenttype/document</property>
+
+  <!-- Hierarchy control -->
+  <property name="allowed_content_types" />
+  <property name="filter_content_types">True</property>
+  <property name="global_allow">True</property>
+
+  <!-- Schema, class and security -->
+  <property name="add_permission">plone.app.contenttypes.addDocument</property>
+  <property name="klass">plone.app.contenttypes.content.Document</property>
+  <property name="model_file">plone.app.contenttypes.schema:document.xml</property>
+  <property name="model_source" />
+  <property name="schema" />
+
+  <!-- Enabled behaviors -->
+  <property name="behaviors"
+            purge="false"
+  >
+    <element value="plone.namefromtitle" />
+    <element value="plone.allowdiscussion" />
+    <element value="plone.excludefromnavigation" />
+    <element value="plone.shortname" />
+    <element value="plone.dublincore" />
+    <element value="plone.richtext" />
+    <element value="plone.relateditems" />
+    <element value="plone.versioning" />
+    <element value="plone.tableofcontents" />
+    <element value="plone.locking" />
+  </property>
+
+  <!-- View information -->
+  <property name="add_view_expr">string:${folder_url}/++add++Document</property>
+  <property name="default_view">document_view</property>
+  <property name="default_view_fallback">False</property>
+  <property name="immediate_view">view</property>
+  <property name="view_methods">
+    <element value="document_view" />
+  </property>
+
+  <!-- Method aliases -->
+  <alias from="(Default)"
+         to="(dynamic view)"
+  />
+  <alias from="edit"
+         to="@@edit"
+  />
+  <alias from="sharing"
+         to="@@sharing"
+  />
+  <alias from="view"
+         to="(selected layout)"
+  />
+
+  <!-- Actions -->
+  <action action_id="view"
+          category="object"
+          condition_expr=""
+          icon_expr="string:toolbar-action/view"
+          title="View"
+          url_expr="string:${object_url}"
+          visible="True"
+          i18n:attributes="title"
+  >
+    <permission value="View" />
+  </action>
+  <action action_id="edit"
+          category="object"
+          condition_expr="not:object/@@plone_lock_info/is_locked_for_current_user|python:True"
+          icon_expr="string:toolbar-action/edit"
+          title="Edit"
+          url_expr="string:${object_url}/edit"
+          visible="True"
+          i18n:attributes="title"
+  >
+    <permission value="Modify portal content" />
+  </action>
+
+</object>
+```
+
+The `name` attribute on the root element in the XML must match the name in the filename and the name listed in `types.xml`.
+
+Set the `i18n:domain` to the i18n domain which includes translations for this content type. This is usually the same as the name of the Python package which contains the content type.
+
+The XML sets a number of FTI properties:
```

Comment:
What's the difference between this list and the next? I think it is meant to be global, correct?

Perhaps a subheading for each list would help clarify. As a bonus, it would provide a visual reference and permalink. When I look at the preview, it visually blurs together as one big list.

```suggestion

(global-fti-properties-label)=

### Global FTI properties

The XML sets a number of FTI properties that are used globally, in both Classic UI and Volto:
```

Reply:
@stevepiercy good idea